hax/hax/types.py

```diff
@@ -159,6 +159,44 @@ class SnsCmStatus(Enum):
     CM_STATUS_PAUSED = 4
 
 
+class m0HaProcessEvent(Enum):
+    M0_CONF_HA_PROCESS_STARTING = 0
+    M0_CONF_HA_PROCESS_STARTED = 1
+    M0_CONF_HA_PROCESS_STOPPING = 2
+    M0_CONF_HA_PROCESS_STOPPED = 3
+
+    def strToEnum(self):
```

Comment:
[Optional]
In python underscore style is more preferred: strToEnum -> str_to_enum

Reply:
Accepted.